### PR TITLE
OSDOCS-10794: Documented the 4.14.28 z-stream release notes

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.14

Issue:
[OSDOCS-10794](https://issues.redhat.com/browse/OSDOCS-10794)

Link to docs preview:
[4.14.28](https://76861--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-28)

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -3319,6 +3319,39 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+[source,terminal]
+----
+$ oc adm release info 4.14.28 --pullspecs
+----
+
+[id="ocp-4-14-28-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Hypershift Operator was not using the `RegistryOverrides` mechanism to inspect the image from the internal registry. With this release, the metadata inspector works as expected during the Hypershift Operator reconciliation, and the `OverrideImages` are properly populated. (link:https://issues.redhat.com/browse/OCPBUGS-33844[*OCPBUGS-33844*])
```
Comment (tmalove) [inline release_notes/ocp-4-14-release-notes.adoc:3342]:
```suggestion
* Previously, the HyperShift Operator was not using the `RegistryOverrides` mechanism to inspect the image from the internal registry. With this release, the metadata inspector works as expected during the HyperShift Operator reconciliation, and the `OverrideImages` are properly populated. (link:https://issues.redhat.com/browse/OCPBUGS-33844[*OCPBUGS-33844*])
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -3319,6 +3319,39 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, the recycler pods for Hosted Control Planes (HCPs) were not starting in disconnected environments. With this release, the HCP recycler-pod image now points to the {product-title} payload reference and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33843[*OCPBUGS-33843*])
+
+* Previously, the information from `imageRegistryOverrides` was only extracted once on the HyperShift Operator initialization and did not refresh. With this release, the Hypershift Operator retrieves the new `ImageContentSourcePolicy` files from the management cluster and adds them to the Hypershift Operator and Control Plane Operator in every reconciliation loop. (link:https://issues.redhat.com/browse/OCPBUGS-33713[*OCPBUGS-33713*])
```
Comment (tmalove) [inline release_notes/ocp-4-14-release-notes.adoc:3346]:
```suggestion
* Previously, the information from `imageRegistryOverrides` was only extracted once on the HyperShift Operator initialization and did not refresh. With this release, the HyperShift Operator retrieves the new `ImageContentSourcePolicy` files from the management cluster and adds them to the HyperShift Operator and Control Plane Operator in every reconciliation loop. (link:https://issues.redhat.com/browse/OCPBUGS-33713[*OCPBUGS-33713*])
```

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -3322,6 +3322,39 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.14.28
+[id="ocp-4-14-28"]
+=== RHSA-2024:3523 - {product-title} 4.14.28 bug fix update and security update
```
Comment (ShaunaDiaz) [inline release_notes/ocp-4-14-release-notes.adoc:3327]:
```suggestion
=== RHSA-2024:3523 - {product-title} 4.14.28 bug fix and security update
```
IMHO don't need "update" twice; not a merge-stopper, just a thought

Comment (dfitzmau) [reply release_notes/ocp-4-14-release-notes.adoc:3327]: Thanks, Shuana. I'll update as part of the 4.14.29 release notes, which I'm also documenting. Thank you!

**Pull request merged**